k8s/hydra/chart/hydra/values.yaml

```diff
@@ -47,8 +47,6 @@ hydra:
   serviceAccount:
     name: hydra-sa
 
-  dangerousForceHttp: true
```

Comment:
Why was this ever a default value?

Reply:
This part was copied from the official chart and for Hydra 1 it meant "Use http instead of https" and it required to work with GKE Ingress. In the Hydra 2 this parameter was replaced by "--dev" arg https://github.com/GoogleCloudPlatform/click-to-deploy/pull/2133/files#diff-ca802c78a4241c1e1e6d68417261298025fa94f66196f9f6d50556133de91dd9R55